docs/developer-docs/latest/development/backend-customization/controllers.md

```diff
@@ -277,6 +273,96 @@ async delete(ctx) {
 ::::
 :::::
 
+::::: details Collection type examples (Replacing a core action)
+
+:::: tabs card
+
+::: tab find()
+
+```js
+async find(ctx) {
+  const { query } = ctx;
+  // some logic here
+  const entity = await strapi.service('api::restaurant.restaurant').find(query);
```

Comment:
I'm probably just nit-picking here but this should be `entities` as it will be more than one